docs/overview/office-add-ins.md

```diff
@@ -1,152 +1,130 @@
-       
-    
-- The permission level and data access requirements for the add-in.
-    
-For more information, see [Office Add-ins XML manifest](../../docs/overview/add-in-manifests.md).
+## Outlook add-ins that extend functionality 
 
+Outlook add-ins can extend the Office ribbon and also display contextually next to an Outlook item when you're viewing or composing it. They can work with an email message, meeting request, meeting response, meeting cancellation, or appointment when a user is viewing a received item or replying or creating a new item. 
 
-###Web app
+Outlook add-ins can access contextual information from the item, such as address or tracking ID, and then use that data to access additional information on the server and from web services to create compelling user experiences. In most cases, an Outlook add-in runs without modification on the various supporting host applications, including Outlook, Outlook for Mac, Outlook Web App, and Outlook Web App for devices, to provide a seamless experience on the desktop, web, and tablet and mobile devices. 
 
-The minimal version of a compliant web app is a static HTML webpage. The page can be hosted on any web server, or web hosting service, such as [Microsoft Azure](../publish/host-an-office-add-in-on-microsoft-azure.md). You can host your web app on the service that you choose.  
+For an overview of Outlook add-ins, see [Outlook add-ins overview](https://docs.microsoft.com/en-us/outlook/add-ins/). 
 
-The most basic Office Add-in consists of a static HTML page that is displayed inside an Office application, but doesn't interact with either the Office document or any other Internet resource. However, because it is a web application, you can use any technologies, both client and server side, that your hosting provider supports (such as ASP.net, PHP, or Node.js). To interact with Office clients and documents, you can use the office.js [JavaScript API](../../docs/develop/understanding-the-javascript-api-for-office.md) that we provide. 
+**Contextual Outlook add-in**
 
+![Contextual add-in](../../images/DK2_AgaveOverview06.png)
 
-**Components of a Hello World Office Add-in**
+## Office JavaScript APIs 
 
-![Components of a Hello World add-in](../../images/DK2_AgaveOverview07.png)
+The Office JavaScript APIs contain objects and members for building add-ins and interacting with Office content and web services. The Excel, Outlook, Word, PowerPoint, OneNote and Project JavaScript APIs provide host-specific object models that you can use in an Office Add-in. These APIs provide access to well-known objects such as paragraphs and workbooks, which makes it easier to create an add-in for a specific host.  
 
-### JavaScript APIs
+To learn more about these APIs, see [Understanding the JavaScript API for Office](../../docs/develop/understanding-the-javascript-api-for-office.md) and the [JavaScript API for Office](../../reference/javascript-api-for-office.md) reference.
```

Comment:
It's probably okay just to remove these references; we already have a link under Additional resources.